### PR TITLE
Add url parameter to preselect tokens

### LINKED ISSUE #266: Add url parameter to preselect output token
In order to have efficient ad targeting, we the output token (and possibly input token) should be pre-selectable through url parameters.

## TODO
* upon startup, check whether there are url parameters `from` or `to`
  * if the url parameter `from` exists, then it will be one of the `InputTokenType` strings
    * if it is, then initially select that input token
  * if the url parameter `to` exists, then it will be one of the `OutputTokenType` strings
    * if it is, then initially select that output token
* if there are no parameters, then the default selection for the input token is `usdc`

### PR DESCRIPTION
**Pull request by TorstenStueber**
Closes #266

### CODE REVIEW

File: src/components/Nabla/schema.tsx [hunk 1]
```diff
@@ -8,8 +8,6 @@ export type SwapFormValues = {
   toAmount: string;
   slippage: number | undefined;
   deadline: number;
-  bankAccount: string;
```
Comment (TorstenStueber) [inline src/components/Nabla/schema.tsx:11]: Just a cleanup. Not required anymore.

File: src/components/Nabla/useSwapForm.tsx [hunk 2]
```diff
@@ -22,17 +22,25 @@ export const useSwapForm = () => {
...
 
-    return {
-      from: (storedFromTokenIsValid ? storageValues?.from : 'usdc') as InputTokenType,
-      to: (storedToTokenIsValid ? storageValues?.to : 'eurc') as OutputTokenType,
-      taxNumber: '',
-      bankAccount: '',
+    const initialValues = {
+      ...defaultValues,
+      ...searchParamValues,
+      ...storageValues,
```
Comment (gianfra-t) [inline src/components/Nabla/useSwapForm.tsx:34]: Shouldn't `searchParamValues` have higher priority than those stored? In case of a redirect.

Comment (TorstenStueber) [reply src/components/Nabla/useSwapForm.tsx:33]: I agree, changed.

**Review by gianfra-t — APPROVED** [review]
✅

**Pull request merged**